recipes/pepr/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
   name: pepr
-  version: "1.1.18"
+  version: "1.1.24"
```

Comment:
Can you please use variables for your version.

Reply:
Is this what you meant?